modules/test/playwright/pages/account-admin-web/EditAccountChannelDefaultsPage.ts

```diff
@@ -25,6 +53,116 @@ export class EditAccountChannelDefaultsPage {
 		this.addDefaultTermSaveButton = page
 			.frameLocator('.fds-modal-body > iframe')
 			.getByRole('button', {name: 'Save'});
+		this.addDefaultShippingAddressButton = page
+			.getByTestId('defaultShippingCommerceAddresses')
+			.getByRole('button', {name: 'Add Default Address'})
+			.first();
+		this.addressTableRowActionsButton = async (
+			tableName: 'billing' | 'shipping',
+			text: string
+		) => {
+			if (tableName === 'billing') {
+				return (
+					await this.getRowByTextFromTable(
+						'defaultBillingCommerceAddresses',
+						text
+					)
+				)
+					.locator('.dnd-td')
+					.nth(3);
+			}
+			else if (tableName === 'shipping') {
+				return (
+					await this.getRowByTextFromTable(
+						'defaultShippingCommerceAddresses',
+						text
+					)
+				)
+					.locator('.dnd-td')
+					.nth(3);
+			}
+			throw new Error(`Invalid table name: ${tableName}`);
+		};
+		this.addressTableRowChannelField = async (
+			tableName: 'billing' | 'shipping',
+			text: string
+		) => {
+			if (tableName === 'billing') {
+				return (
+					await this.getRowByTextFromTable(
+						'defaultBillingCommerceAddresses',
+						text
+					)
+				)
+					.locator('.dnd-td')
+					.nth(0);
+			}
+			else if (tableName === 'shipping') {
+				return (
+					await this.getRowByTextFromTable(
+						'defaultShippingCommerceAddresses',
+						text
+					)
+				)
+					.locator('.dnd-td')
+					.nth(0);
+			}
+			throw new Error(`Invalid table name: ${tableName}`);
+		};
+		this.addressTableRowNameField = async (
+			tableName: 'billing' | 'shipping',
+			text: string
+		) => {
+			if (tableName === 'billing') {
+				return (
+					await this.getRowByTextFromTable(
+						'defaultBillingCommerceAddresses',
+						text
+					)
+				)
+					.locator('.dnd-td')
+					.nth(1);
+			}
+			else if (tableName === 'shipping') {
+				return (
+					await this.getRowByTextFromTable(
+						'defaultShippingCommerceAddresses',
+						text
+					)
+				)
+					.locator('.dnd-td')
+					.nth(1);
+			}
```

Comment:
and also here

and because they are the same but the `.nth(number)` can we pass it as argument so this can be only one method?